### PR TITLE
Update clientcli.rst

### PR DESCRIPTION
**Pull request by btl5037**

### CODE REVIEW

File: docs/source/clientcli.rst [hunk 1]
```diff
@@ -41,7 +41,7 @@ Fabric-CA Client's CLI
           --id.name string                 Unique name of the identity
           --id.secret string               The enrollment secret for the identity being registered
           --id.type string                 Type of identity being registered (e.g. 'peer, app, user') (default "client")
-          --loglevel string                Set logging level (info, warning, debug, error, fatal, critical)
+          --loglevel string                Set loggi ng level (info, warning, debug, error, fatal, critical)
```
Comment (btl5037) [inline docs/source/clientcli.rst]: Why are we adding a space here

Comment (nikhil550) [reply docs/source/clientcli.rst]: If its what you say I love it, especially later in the summer

**Pull request closed without merging**